src/FSharp.Build/Microsoft.FSharp.NetSdk.props

```diff
@@ -73,6 +73,13 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
     <PackageReference Include="System.ValueTuple" Version="$(ValueTupleImplicitPackageVersion)" />
   </ItemGroup>
 
+  <PropertyGroup Condition=" '$(ManagePackageVersionsCentrally)' == 'true' ">
+    <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+    <DisableImplicitLibraryPacksFolder>true</DisableImplicitLibraryPacksFolder>
+    <_FSCorePackageVersionSet>true</_FSCorePackageVersionSet>
+    <_FSharpCoreLibraryPacksFolder></_FSharpCoreLibraryPacksFolder>
```

Comment:
Is there a need for such changes?

Reply:
Probably not. We just need not to load the package itself.